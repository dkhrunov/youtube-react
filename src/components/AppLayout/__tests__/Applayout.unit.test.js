import React from 'react';
import { shallow } from 'enzyme';
import AppLayout from '../AppLayout';

test('renders empty <AppLAyout />', () => {
	const wrapper = shallow(<AppLayout />);
	expect(wrapper).toMatchSnapshot();
});

test('renders <AppLAyout /> with one child', () => {
	const wrapper = shallow(
		<AppLayout>
			<div>Child 1</div>
		</AppLayout>
	);
	expect(wrapper).toMatchSnapshot();
});

test('renders <AppLAyout /> with childrens', () => {
	const wrapper = shallow(
		<AppLayout>
			<div>Child</div>
			<div>
				<span>Child</span>
				<p>Child</p>
			</div>
		</AppLayout>
	);
	expect(wrapper).toMatchSnapshot();
});
