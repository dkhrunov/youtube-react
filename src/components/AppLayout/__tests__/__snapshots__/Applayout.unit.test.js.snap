// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <AppLAyout /> with childrens 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    Child
  </div>
  <div>
    <span>
      Child
    </span>
    <p>
      Child
    </p>
  </div>
</div>
`;

exports[`renders <AppLAyout /> with one child 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
  <div>
    Child 1
  </div>
</div>
`;

exports[`renders empty <AppLAyout /> 1`] = `
<div
  className="app-layout"
>
  <HeaderNav />
</div>
`;
